runtime/src/main/java/org/corfudb/runtime/LogReplicationLogicalGroupClient.java

```diff
@@ -143,6 +143,9 @@ private void register() {
 
     /**
      * Adds a destination to LogReplicationModelMetadataTable for monitoring.
```

Comment:
I think this method description should be modified, the description "Adds a destination to LogReplicationModelMetadataTable for monitoring." is telling about what internally this means to LR, instead it should point to what is relevant for the client/consumer which does not require knowledge on LR internals. It should be something like:

"Add/associate a destination to a given logical group"